components/HomeComponents/about-the-clinic.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { PageType } from '../Types'
+import { Button } from '../dataComponents/button'
+
+export default function AboutTheClinic({ data }: PageType) {
+  return (
+    <>
+      <section className="grid md:grid-cols-2 md:gap-4 ">
+        <div className="hidden md:block w-[600px] h-[500px] mt-6 mr-20 border bg-[url('')]"></div>
```

Comment:
You can't set a div to have width 600px because what will happen when the screen size is 500px for example? It will overflow the screen as you can see in the image, the image is going over the text when the screen is narrow.
![Screen Shot 2022-02-27 at 15 40 07](https://user-images.githubusercontent.com/13974474/155884858-4cb622ce-bdcf-40bf-9a2b-584e2836fe55.png)
.

Reply:
all of w-[any size] style has removed from the project
replaced it with max/min-h/w